{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/util/templatetags/filters.py

```diff
@@ -0,0 +1,29 @@
+import json
+
+from django import template as template_lib
+
+# The below imports should be removed when Django 4.1 releases
+from django.utils.html import _json_script_escapes, format_html
+from django.utils.safestring import mark_safe
+
+register = template_lib.Library()
+
+
+# BACKPORT: Remove once Django 4.1 releases
+@register.filter(is_safe=True)
+def json_script_noid(value, element_id=None):
+    """
+    Escape all the HTML/XML special characters with their unicode escapes, so
```

Comment:
Maybe it's just worth including an example and what it is escape to  at the bottom in case someone is unfamiliar, e.g. 

```py
    """
    Escape all the HTML/XML special characters with their unicode escapes, so
    value is safe to be output anywhere except for inside a tag attribute. Wrap
    the escaped JSON in a script tag.
    
    e.g. “ -> &quot; 
          & -> &amp;
    etc.
    """

Reply:
This is a just a temp backport of `json_script` from Django's main branch (see django/django#15103), it'll be removed and references to it replaced with `json_script` with the next point release. Maybe we should link to the docs for that filter in the comment above the function?

Reply:
That's okay, it's just that do later often becomes do never, so whichever project gets this in the mean time will likely be stuck with this for some time.

Reply:
That's a great point. Do you have any strong opinions on how to handle that?